stable/metacontroller/templates/rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if .Values.rbac.create }}
+{{ if .Values.serviceAccount.create }}
```

Comment:
this does not depend on serviceAccount.create - only rbac.create, see https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md